src/screens/Books.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import Book from '../components/Book';
+import Form from '../components/Form';
+
+const Books = () => {
+    return (
+        <>
+            <ul className='max-w-[1300px] pr-10 h-[calc(100vh-250px)] books-container flex flex-col gap-4  overflow-auto  border m-auto mt-10'>
+                <Book />
+            </ul>
+            <Form />
+        </>
+    )
+}
+
+export default Books
```

Comment:
please note that your `Book ` component should take the book data as a prop and renders the title and the author, so your book component should have title and author as props 😄

Reply:
well resolved! ✔️